src/app/containers/Figure/index.jsx

```diff
@@ -78,15 +79,47 @@ FigureContainer.propTypes = {
   type: string,
   srcset: string,
   width: number.isRequired,
+  showCopyright: bool,
 };
 
-FigureContainer.defaultProps = {
+const defaultProps = {
   copyright: null,
   captionBlock: null,
   height: null,
   lazyLoad: false,
   type: '',
   srcset: null,
+  showCopyright: false,
+};
+
+ImageComponent.propTypes = {
+  ...figurePropTypes,
+  platform: oneOf(['amp', 'canonical']).isRequired,
+};
+
+ImageComponent.defaultProps = { ...defaultProps };
+
+const FigureContainer = props => {
+  const { platform } = useContext(RequestContext);
+  const { useFigure } = props;
+
+  return useFigure ? (
+    <Figure>
+      <ImageComponent {...props} platform={platform} />
```

Comment:
This change will lead to the Figure Container to optionally not include the `<figure>` element.  
We want to make this Figure container a reusable component - to move it into Psammead as a Psammead Figure Container.  I think this could be quite a confusing change. 

If this path is still desired, I suggest writing this change in a different way, to reduce the cognitive complexity of having two containers with different return statements.

```
const Wrapper = useFigure ? Figure : Fragment; 
```

Then keep a single container in this file, with 
```
  return (
    <Wrapper>
     .....
    </Wrapper>
  );
```

Reply:
+1 for `const Wrapper = useFigure ? Figure : Fragment;` :)